src/DataTable/index.jsx

```diff
@@ -187,6 +187,7 @@ function DataTable({
 
   const enhancedInstance = {
     ...instance,
+    fetchData,
```

Comment:
I think we should look at `manualFilters` instead, if it is set to `true` it means that we perform filtering ourselves (e.g. via request to backend), that's how react-table knows not to perform frontend filtering on its own (see [docs](https://github.com/TanStack/table/blob/v7/docs/src/pages/docs/api/useFilters.md)).

Reply:
Corrected, thanks 👍